src/com.frostycoolslug.goxlr-utility.sdPlugin/app/display-volume.js

```diff
@@ -114,7 +115,44 @@ class VolumeMonitor {
 
         let percent = Math.round(((volume / 255) * 100));
 
-        let svg = `<svg height="100" width="100"><text fill="#fff" font-family="Monospace" font-size="26px" x="50&#37;" y="50&#37;" dominant-baseline="middle" text-anchor="middle">${percent}&#37;</text></svg>`;
+        let muteStatus = '';
+        let muteTarget = '';
+        for (const [key, value] of Object.entries(faders)) {
+            if (value.channel === this.channel_a) {
+                if (value.mute_state !== "Unmuted") {
+                    muteStatus = "MUTED"
+                    if (value.mute_state === "MutedToX") {
+                        switch (value.mute_type) {
+                            case "ToStream":
+                                muteTarget = "Stream";
+                                break;
+                            case "ToChat":
+                                muteTarget = "Chat";
+                                break;
+                            case "ToHeadphones":
+                                muteTarget = "Headphones";
+                                break;
+                            case "ToLineOut":
+                                muteTarget = "Line Out";
+                                break;
+                        }
+                    } else {
+                        muteTarget = "GLOBAL";
```

Comment:
wondering if I should remove this, "Mute" and "Mute Global" are effectively the same thing
one is ->MuteToX(All) the other is ->MuteToAll

Reply:
The only real difference between the two as far as the GoXLR is concerned, is that one will be a solid colour, and the other will flash, under the hood the mute behaviours are the same.. So I'll happily leave that one up to you :)